### PR TITLE
[Eslint] Enabled Eslint and fixed up styles

### PR DESCRIPTION
**Pull request by franklixuefei**
Enabled Eslint and fixed up styles, as part of #197 , using `npm run lint` and `npm run lint-fixup`

### CODE REVIEW

**Comment by franklixuefei** [comment]
No code changes. Just styling change.

File: src/js/themes/styleConstants.js [hunk 1]
```diff
@@ -1,101 +1,96 @@
...
-  undefinedBorderRadius: '3px',
+    undefinedFontSize: '11px',
+    undefinedFontWeight: 'bold',
+    undefinedPadding: '1px 2px',
+    undefinedBorderRadius: '3px',
 
-  nullFontSize: '11px',
-  nullFontWeight: 'bold',
-  nullPadding: '1px 2px',
-  nullBorderRadius: '3px',
```
Comment (franklixuefei) [inline src/js/themes/styleConstants.js]: I removed these four lines because these are dupes of line 43-46

**Comment by mac-s-g** [comment]
love it.  testing locally now.

**Comment by mac-s-g** [comment]
tests passing :+1:

**Pull request merged**